samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,178 @@
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+
+   In addition, button 1 is used to simulate presence detected.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/sensor_server`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_sensor_server_testing:
+
+Testing
+=======
+
+.. important::
+   The :ref:`bt_mesh_sensor_srv_readme` sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_sensor_client` sample running in the same Mesh network.
+   Before testing Mesh :ref:`bt_mesh_sensor_srv_readme`, go through the Mesh :ref:`bt_mesh_sensor_cli_readme`'s :ref:`bluetooth_mesh_sensor_client_testing` guide with a different board.
+
+After programming the sample to your board, you can test the device by provisioning and configuring it for communication trough the Mesh models.
+This can be done using a smartphone with Nordic Semiconductor's nRF Mesh app installed.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the Mesh network.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned Mesh devices.
+#. Select the :guilabel:`Mesh Sensor Server` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select an OOB method and follow the instructions in the app.
+
+Once the provisioning is complete, the app returns to the Network screen.
+
+.. _bluetooth_mesh_sensor_server_conf_models:
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Sensor Server` node.
+   Basic information about the Mesh node and its configuration is displayed.
+#. In the Mesh node view, expand the element.
+   It contains the list of models in the first and only element of the node.
+#. Tap :guilabel:`Sensor Server` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   1. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+#. Set the publishing parameters:
+
+   1. Tap :guilabel:`SET PUBLICATION`.
+   #. Tap :guilabel:`Publish Address`.
+   #. Choose Groups from the dropdown menu.
+   #. Choose an existing group or create a new.
+         The Sensor Client must subscribe to the same grop.
+   #. Tap :guilabel:`OK`.
+   #. Choose a publishing period by using the :guilabel:`Interval` slider.
+   #. Retransmit Count should be 0 to avoid duplicate logging in the :ref:`bt_mesh_sensor_cli_readme` UART terminal.
+   #. Press the :guilabel:`check button` at the bottom right corner.
```

Comment:
```suggestion
   #. Retransmit Count should be 0 to avoid duplicate logging in the :ref:`sensor client's <bt_mesh_sensor_cli_readme>` UART terminal.
   #. Tap the :guilabel:`check button` at the bottom right corner.
```